### ISSUE TITLE
Suggestion:  Calculate Sunrise/Sunset

### ISSUE THREAD

**Issue by jkms** [Fushko/gammy#17]
First of all, Gammy is great!  Love that it actually adjusts the brightness of my ancient LCD's!

It would be nice if I could enter my coordinates, and have the times adjust automagically.

I'm sure you've seen this feature before, so I won't waste too much effort describing it.

Thanks!

**Comment by Fushko** [comment]
That's already planned. Don't hold your breath for it, I have limited time to work on it right now.

**Comment by Gitoffthelawn** [comment]
Just a note for the future... if this is implemented, please include separate "skew" values for both sunrise and sunset.

For example, allow the user to specify -0:30 to be applied to sunrise, to indicate to end the transition to daytime 30 minutes after sunrise.  Thus if sunrise is calculated to be 6am, complete the transition to daytime values by 5:30am.

Similarly, the user could specify +1:00 to be applied to sunset, to indicate to end the transition to nighttime by 1 hour after sunset. Thus if sunset is calculated to be 5pm, complete the transition to nighttime by 6pm.

**Comment by Fushko** [comment]
> Just a note for the future... if this is implemented, please include separate "skew" values for both sunrise and sunset.
> 
> For example, allow the user to specify -0:30 to be applied to sunrise, to indicate to end the transition to daytime 30 minutes after sunrise. Thus if sunrise is calculated to be 6am, complete the transition to daytime values by 5:30am.
> 
> Similarly, the user could specify +1:00 to be applied to sunset, to indicate to end the transition to nighttime by 1 hour after sunset. Thus if sunset is calculated to be 5pm, complete the transition to nighttime by 6pm.

Makes sense. I imagine having the lowest temperature exactly at sunset wouldn't always be preferable.
